pypowsybl/network.py

```diff
@@ -214,75 +214,78 @@ def get_elements_ids(self, element_type: _pypowsybl.ElementType, nominal_voltage
                                                    main_connected_component, main_synchronous_component,
                                                    not_connected_to_same_bus_at_both_sides)
 
-    def get_elements(self, element_type: _pypowsybl.ElementType) -> pd.DataFrame:
+    def get_elements(self, element_type: _pypowsybl.ElementType, bus_breaker_view= False) -> pd.DataFrame:
         """ Get network elements as a ``Pandas`` data frame for a specified element type.
 
         Args:
             element_type (ElementType): the element type
         Returns:
             a network elements data frame for the specified element type
         """
-        series_array = _pypowsybl.create_network_elements_series_array(self._handle, element_type)
+        series_array = _pypowsybl.create_network_elements_series_array(self._handle, element_type, bus_breaker_view)
         return create_data_frame_from_series_array(series_array)
 
-    def get_buses(self) -> pd.DataFrame:
-        return self.get_elements(_pypowsybl.ElementType.BUS)
+    def get_buses(self, bus_breaker_view=False) -> pd.DataFrame:
```

Comment:
With that new argument to all methods, I'm afraid we need to update all the docstrings!!